airflow/providers/amazon/aws/sensors/rds.py

```diff
@@ -61,7 +64,9 @@ def _check_item(self, item_type: str, item_name: str) -> bool:
         except ClientError:
             return False
         else:
-            return bool(items) and any(map(lambda s: items[0]['Status'].lower() == s, self.target_statuses))
+            return bool(items) and any(
+                map(lambda s: items[0][self.check_status_field].lower() == s, self.target_statuses)
```

Comment:
Nitpick:  Can you rename `s` here to something useful?

Reply:
Thanks, updated in https://github.com/apache/airflow/pull/26003/commits/82813173d05782562dfae3b68394df3c8082d03a